GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -478,6 +491,45 @@ private int safeCompare(Date date1 , Date date2 ) {
         log.debug("Finished data sort");
     }
 
+    //gather some statistics on the data collection
+    private void analyseData(final List<InstanceData> allData) {
+        totalInstances = allData.size();
+        
+        for (InstanceData instance : allData) {
+            SurveyInstanceDto sid = instance.surveyInstanceDto;
```

Comment:
why not call it survyeInstanceDto?

Reply:
Too long to follow easily. We usually do this for loop variables.